### PR TITLE
V2 config

### LINKED ISSUE #800: Edit user profile
Fields to edit:

* email
* language 
* template
* entries per page
* ...

### PR DESCRIPTION
**Pull request by j0k3r**
The config page:

 - [x] Basic config (theme, items per page, language)
 - [x] Changing password
 - [x] Updating logged in user (email, name, etc ..)
 - [x] Adding new user

I also improved InstallCommand & add tests

Will fix #800 
Related #673

### CODE REVIEW

File: src/Wallabag/CoreBundle/Command/InstallCommand.php [hunk 1]
```diff
@@ -135,21 +135,13 @@ protected function setupAdmin(OutputInterface $output)
...
         $em->persist($user);
 
-        $pagerConfig = new UsersConfig();
-        $pagerConfig->setUserId($user->getId());
-        $pagerConfig->setName('pager');
-        $pagerConfig->setValue(10);
+        $config = new Config();
+        $config->setUser($user);
+        $config->setTheme('baggy');
+        $config->setItemsPerPage(10);
```
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Command/InstallCommand.php]: 12 is the new default value ;-)

Comment (tcitworld) [inline src/Wallabag/CoreBundle/Command/InstallCommand.php]: Shouldn't there be a config file for default values ?

Comment (j0k3r) [inline src/Wallabag/CoreBundle/Command/InstallCommand.php]: Yes, that's the purpose of `parameters.yml.dist`. I didn't use it atm, but we'll have to in the future.

**Comment by nicosomb** [comment]
I wonder if theme / language / etc have to be stored in config table or user table.

File: src/Wallabag/CoreBundle/Command/InstallCommand.php (hunk 1 above)
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Command/InstallCommand.php]: Oh, I'm sorry, didn't saw the file.

**Comment by tcitworld** [comment]
Hmm, tricky question. If we put them in the user table, I guess we will quickly have too much columns.

**Comment by j0k3r** [comment]
I asked myself the same question. I ended up using a different table to separate such information and avoid to much noise in the user table. But we can change it

**Comment by nicosomb** [comment]
That's why I asked you ;-)

**Comment by j0k3r** [comment]
@nicosomb if everything seems ok to you, it's ready to merge :+1:

**Comment by nicosomb** [comment]
I switched on your branch, removed my database, executed `app/console wallabag:install`, followed the steps, and I've got this error:
```
  [Symfony\Component\DependencyInjection\Exception\InvalidArgumentException]
  The parameter "theme" must be defined.
```
and when I tried to connect myself, `bad credentials`.

**Comment by j0k3r** [comment]
I've added some new parameters in `parameters.yml.dist`, a `composer install` should solve your problem

**Comment by nicosomb** [comment]
It works :+1:

**Comment by nicosomb** [comment]
All seems good for me. Still in WIP or not?

**Comment by j0k3r** [comment]
No more :)

**Pull request merged**